Sources/SwiftSyntax/generated/TokenKind.swift

```diff
@@ -1,5 +1,19 @@
 
 
+//// Automatically Generated by generate-swiftsyntax
```

Comment:
Not part of this PR but do you know what generates these two newlines at the start of each file?

Reply:
I have no clue! Other files it's just one new line

Reply:
I can investigate

Reply:
I think none of them should contain a leading newline before the comment. If you could investigate, that’d be great but as I said, it’s not a top priority.